.travis.yml

```diff
@@ -42,7 +42,7 @@ script:
 - ./gradlew check -S --info --console plain
 
 after_success:
-- bash <(curl -s https://codecov.io/bash) -F unittest
+- test $TRAVIS_BRANCH = "master" && bash <(curl -s https://codecov.io/bash) -F unittest
```

Comment:
Should change this to check for `CODE_COV_TOKEN` later. But it works for our purposes meanwhile.